data/products.yml

```diff
@@ -35,7 +35,7 @@ influxdb3_explorer:
   menu_category: tools
   list_order: 1
   latest: explorer
-  latest_patch: 1.0.0
+  latest_patch: 1.1.0
```

Comment:
I have this in the [Explorer 1.1 PR](https://github.com/influxdata/docs-v2/pull/6235).